### PR TITLE
Added login page and react-router

### PR DESCRIPTION
**Pull request by bornagojsic**
- also added dark theme in index.css and fixed components.json

### CODE REVIEW

**Comment by stjepanbonic** [comment]
can you put some screenshots in the description of the PR so we can see how it looks?

**Comment by stjepanbonic** [comment]
also we will need to check if the page should be in Croatian or in English

File: frontend/src/routes/pathConstants.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const PathConstants = {
+    HOME: "/",
+    LOGIN: "/login",
+    SIGNUP: "/signup",
```
Comment (stjepanbonic) [inline frontend/src/routes/pathConstants.ts:4]: I'd rather use /register here, or change /login to /signin, I'd rather have register and login tho

Comment (bornagojsic) [reply frontend/src/routes/pathConstants.ts:4]: Ok, I'll change it

**Review by stjepanbonic — APPROVED** [review]

**Pull request merged**